modules/library/main/src/test/java/org/geotools/styling/visitor/RenderingSelectorStyleVisitorTest.java

```diff
@@ -0,0 +1,111 @@
+package org.geotools.styling.visitor;
+
+import static org.geotools.styling.visitor.RenderingSelectorStyleVisitor.RENDERING_LEGEND_OPTION;
+import static org.geotools.styling.visitor.RenderingSelectorStyleVisitor.RENDERING_MAP_OPTION;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+import org.geotools.factory.CommonFactoryFinder;
+import org.geotools.styling.FeatureTypeStyle;
+import org.geotools.styling.Rule;
+import org.geotools.styling.Style;
+import org.geotools.styling.StyleBuilder;
+import org.geotools.styling.StyleFactory;
+import org.geotools.styling.Symbolizer;
+import org.junit.Before;
+import org.junit.Test;
+import org.opengis.filter.FilterFactory2;
+
+public class RenderingSelectorStyleVisitorTest {
+
+    StyleBuilder sb;
+    StyleFactory sf;
+    FilterFactory2 ff;
+
+    @Before
+    public void setUp() throws Exception {
+        sf = CommonFactoryFinder.getStyleFactory(null);
+        ff = CommonFactoryFinder.getFilterFactory2(null);
+        sb = new StyleBuilder(sf, ff);
+    }
+
+    @Test
+    public void testMapRenderingSelection() {
+        Style oldStyle = sb.createStyle("FTSName", sf.createPolygonSymbolizer());
```

Comment:
It's not really "old", it's the initial style. I'd simply call it 
```suggestion
        Style style = sb.createStyle("FTSName", sf.createPolygonSymbolizer());
```

Reply:
fixed